static/templates/navbar_alert.hbs

```diff
@@ -0,0 +1,50 @@
+<div data-process="{{data_process}}" class="alert alert-info {{class_name}}">
+    <div data-step="1">
+        {{#if is_notification_alert}}
+        {{> notification_alert_first_step_content}}
+        {{else if is_bankruptcy_alert}}
```

Comment:
My plan here was just to use the `first_step_alert_content` variable (instead of using `is_notification_alert` and `is_bankruptcy_alert`) which will lead to a handlebar file that has the content.

But I couldn't find any relevant helper for it, then I thought I'd create a manual helper but for it, but I can't as we need to read file content and the `fs` module is not allowed on the client-side.